packages/amplify-environment-parameters/package.json

```diff
@@ -15,6 +15,9 @@
     "amplify",
     "aws"
   ],
+  "publishConfig": {
```

Comment:
How is this change related ?

Reply:
The package needs to be marked as `public` when publishing or NPM will return a 402 error because private packages are a paid service (and they are the default publishing mode).

Reply:
I think such changes should be different PR to avoid situations where this change had to be reverted.

Reply:
It was a different PR but git shows it as diff here after we force pushed branch when fixing version mess.
https://github.com/aws-amplify/amplify-cli/pull/11029